packages/command/src/sfdxFlags.ts

```diff
@@ -161,7 +175,7 @@ function buildOption<T>(
 }
 
 function buildString(options: flags.String): flags.Discriminated<flags.String> {
-  return merge('string', OclifFlags.string(options), options);
+  return merge<string>('string', OclifFlags.string(options), options);
```

Comment:
I think this type argument is required which makes sense that the resulting type of the parse function for the flag is a `string` for the `buildString` function.